geode-examples/README.md

```diff
@@ -0,0 +1,70 @@
+# Apache Geode examples
+
+This is the home of Apache Geode examples that are bundled with the project. Contributions<sup>[2]</sup> and corrections are welcome. Please talk to us about your suggestions at [dev@geode.incubator.apache.org](mailto:dev@geode.incubator.apache.org) or submit a [pull request](https://github.com/apache/incubator-geode/pull/new/develop).
+
+## Example requirements
+
+All examples:
+
+*  Need to be testable. Use unit tests, integration tests or whatever is applicable. Tests will run through the project's CI.
+*  Should be `Gradle` projects or part of existing ones. There may be exceptions here, but the community should have a consensus to accept.
+*  Have to follow code format & style from Apache Geode <sup>[1]</sup> guidelines.
+*  Should contain a `README.md` file with step-by-step instruction on how to set up and run the example. *Diagrams give you extra credit.*
+*  Donations need to be licensed through ASL 2.0 and contributors need to file an ICLA<sup>[3]</sup>.
+
+## Structure
+
+### Installation and a Tutorial for Beginners
+
+*  [How to Install](http://geode.docs.pivotal.io/docs/getting_started/installation/install_standalone.html)
+*  Set a `GEODE_HOME` environment variable to point to the `bin` directory of the installation. For those that have built from source, this will be the `geode-assembly/build/install/apache-geode` directory.
```

Comment:
Seems like the language is unclear here: Set a `GEODE_HOME` environment variable to point to the `bin` directory

Pointing to 'bin' would not be correct as all the scripts would not work. I think wording should be changed to: 'root' directory of the installation.